tingbot/run_loop.py

```diff
@@ -109,11 +111,27 @@ def add_before_action_callback(self, callback):
 
     def add_after_action_callback(self, callback):
         self._after_action_callbacks.add(callback)
-
+        
+    @classmethod
+    def callAfter(cls, func):
+        cls.callAfterQueue.put(func)
+        
+    def clearQueue(cls):
+        while True:
+            try:
+                func = cls.callAfterQueue.get_nowait()
+                func()
+                cls.callAfterQueue.task_done()
+            except Queue.Empty:
+                break
```

Comment:
I think this should also be an @classmethod. Also can we rename to something a bit more explicit, like `emptyCallAfterQueue`?